src/pages/Bridge/index.tsx

```diff
@@ -132,25 +193,39 @@ const BridgePage: React.FC = () => {
     const formValues = getValues();
 
     handleChainFieldChange({
-      fromChainId: formValues.toChainId,
-      toChainId: formValues.fromChainId,
+      newFromChainId: formValues.toChainId,
+      newToChainId: formValues.fromChainId,
     });
   }, [getValues, handleChainFieldChange]);
 
-  const onSubmit = (values: z.infer<typeof formSchema>) => {
-    // TODO: send transaction (see VEN-2248)
-    console.log(values);
-  };
+  const { mutateAsync: bridgeXvs } = useBridgeXvs();
 
-  // TODO: wire up
-  const walletBalanceTokens = new BigNumber(10);
-  const limitTokens = walletBalanceTokens;
+  const onSubmit = () => {
+    bridgeXvs({
+      accountAddress: accountAddress || '',
+      amountMantissa,
+      destinationChain: toChainId,
+      gasValueMantissa: bridgeEstimatedFeeMantissa,
+    });
+  };
```

Comment:
To prevent ever calling the bridge with an empty address: 
```suggestion
  const onSubmit = () => {
    if (accountAddress) {
      bridgeXvs({
        accountAddress,
        amountMantissa,
        destinationChain: toChainId,
        gasValueMantissa: bridgeEstimatedFeeMantissa,
      });
    }
  };
```